Types/ELF/include/elf.hpp

```diff
@@ -0,0 +1,419 @@
+#pragma once
+
+#include "utils.hpp"
+
+namespace GView::Type::ELF
+{
+namespace Go
+{
+    constexpr auto ELF_GO_BUILD_ID_TAG = 4U;
+    constexpr auto GNU_BUILD_ID_TAG    = 3U;
+
+    constexpr auto ELF_GO_NOTE  = std::string_view("Go\x00\x00", 4);
+    constexpr auto ELF_GNU_NOTE = std::string_view("GNU\x00", 4);
+
+    // version of the pclntab (Program Counter Line Table) -: https://go.dev/src/debug/gosym/pclntab.go
+    enum class PclntabVersion : int32
+    {
+        Unknown = -1,
+        _11     = 0,
+        _12     = 1,
+        _116    = 2,
+        _118    = 3,
+    };
+
+    enum class GoMagic : uint32 // https://go.dev/src/debug/gosym/pclntab.go
+    {
+        _12  = 0xfffffffb,
+        _116 = 0xfffffffa,
+        _118 = 0xfffffff0,
+    };
+
+    static std::string_view GetNameForGoMagic(GoMagic magic)
```

Comment:
nu e mai usor sa fie numere direct astea?

Reply:
e de fapt 1.2, 1.16, 1.18, cred
ramane de vazut.. tre' sa ma mai uit in directia asta
https://github.com/gdt050579/GView/issues/68